src/snowflake/snowpark/dataframe.py

```diff
@@ -277,15 +277,15 @@ def _disambiguate(
             )
             for name in lhs_names
         ],
```

Comment:
this change makes sense to me, let's also check that all docstrings are updated with a unique description (that probably needs to get copied around). If you dislike copy&paste of the docstring, we could think of a decorator to inject a unique description. For mid-term, we may need to think about using a PIMPL pattern for the actual implementation. Most of the current Snowpark APIs are public, so adding a new parameter could be problematic.

Reply:
Updated all docstrings to use the same description.